algorithms/kernel/logistic_regression/logistic_regression_predict_types.cpp

```diff
@@ -93,49 +91,11 @@ services::Status Input::check(const daal::algorithms::Parameter *parameter, int
     const daal::algorithms::logistic_regression::internal::ModelImpl* pModel =
         static_cast<const daal::algorithms::logistic_regression::internal::ModelImpl*>(m.get());
     DAAL_ASSERT(pModel);
-    const daal::algorithms::logistic_regression::prediction::Parameter* pPrm = static_cast<const daal::algorithms::logistic_regression::prediction::Parameter*>(parameter);
+    const daal::algorithms::logistic_regression::prediction::interface1::Parameter* pPrm = static_cast<const daal::algorithms::logistic_regression::prediction::interface1::Parameter*>(parameter);
```

Comment:
Dependency on deprecated 'interface1::Parameter' inside still valid interface1::Input which is used in interface2::Batch.

Reply:
Fixed